src/routes/AppContent/index.tsx

```diff
@@ -82,7 +90,7 @@ const AppContent = () => {
     // Check if the account list currencies are in the client accounts currencies which is taken from OIDC tokens
     // If not, request OIDC authentication
     useEffect(() => {
-        if (accountList && isOAuth2Enabled) {
+        if (accountList?.length > 0 && isOAuth2Enabled) {
```

Comment:
when checking if accountList is there, an empty array will return true, so need to check the length instead